cloud_testing/kotlin_poc/src/main/kotlin/ftl/run/IosTestRunner.kt

```diff
@@ -23,21 +24,30 @@ object IosTestRunner : GenericTestRunner {
         val iosDevice = IosDevice()
                 .setIosModelId(IosCatalog.model("iphone8"))
                 .setIosVersionId(IosCatalog.version("11.2"))
-                .setLocale("en_US") // iOS doesn't support other locales or orientations
+                .setLocale("en_US") // FTL iOS doesn't currently support other locales or orientations
```

Comment:
true, what about model versions? we should be able to configure that right away.

Reply:
Agreed, I opened a ticket:
https://github.com/TestArmada/flank/issues/193

I'm hesitant to add new features until we comprehensively test the existing code. That's being worked on in https://github.com/TestArmada/flank/pull/190